docs/guides/expectations/advanced/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -9,7 +9,7 @@ In this tutorial, you will develop hands-on experience with configuring a Rule-B
 
 - Have a basic understanding of [Metrics in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/metrics.html)
 - Have a basic understanding of [Expectation Configurations in Great Expectations](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/expectations.html#expectation-concepts-domain-and-success-keys)
```

Comment:
Just wanted to double-check. The reason why this still points to `docs.greatexpectations.io` is because the equivalent DOCS have not been migrated onto the new site? 

- If there isn't one already, could you make a JIRA task to make sure that we (eventually) migrate all the docs to the new site?

Reply:
@Shinnnyshinshin - Yup! Thanks for the reminder - will do! Should I make a broader task for migrating all the docs, or specific tasks for Metrics and Expectations?